gradle-examples/jaxrs-cdi/build.gradle

```diff
@@ -53,7 +53,6 @@ dependencies {
   implementation "commons-io:commons-io:2.4"
 
   testImplementation "junit:junit:4.12"
-  testImplementation "org.gradle:gradle-tooling-api:${gradle.gradleVersion}"
```

Comment:
Since we are pulling the tooling API via the plugin, we no longer need to specify it as an explicit dependency.